src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -735,14 +737,107 @@ def check_server_name_availability(check_name_client, server_name, service_name)
     return True
 
 
-def update_local_contexts(cmd, server_name, resource_group_name, location, user):
+def update_local_contexts(cmd, provider, server_name, resource_group_name, location, user):
+    engine = 'postgres'
+    if provider == 'Microsoft.DBforMySQL':
+        engine = 'mysql'
+    elif provider == 'Microsoft.DBforMariaDB':
+        engine = 'mariadb'
+
     if cmd.cli_ctx.local_context.is_on:
-        cmd.cli_ctx.local_context.set(['postgres server'], 'server_name',
+        cmd.cli_ctx.local_context.set([engine], 'server_name',
                                       server_name)  # Setting the server name in the local context
-        cmd.cli_ctx.local_context.set(['postgres server'], 'administrator_login',
+        cmd.cli_ctx.local_context.set([engine], 'administrator_login',
                                       user)  # Setting the server name in the local context
         cmd.cli_ctx.local_context.set([ALL], 'location',
                                       location)  # Setting the location in the local context
         cmd.cli_ctx.local_context.set([ALL], 'resource_group_name', resource_group_name)
 
-# end region
+
+def get_connection_string(cmd, client, server_name='{server}', database_name='{database}', administrator_login='{username}', administrator_login_password='{password}'):
+    provider = 'MySQL'
+    if isinstance(client, PostgreSQLLocationOperations):
+        provider = 'PostgreSQL'
+    elif isinstance(client, MariaDBLocationOperations):
+        provider = 'MariaDB'
+
+    if provider == 'MySQL':
+        host = '{}.mysql.database.azure.com'.format(server_name)
```

Comment:
host may be changed according to cloud, right?

Reply:
Zunli, I don't think I understood this correctly. Do you mean there should be change in defining host here...?

Reply:
yes. If different cloud will have different suffix. https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/cloud.py#L316 for your reference

Reply:
Thanks! Fixed!